homeassistant/components/tado/sensor.py

```diff
@@ -225,6 +226,9 @@ def _async_update_zone_data(self):
             self._state = self._tado_zone_data.open_window
             self._state_attributes = self._tado_zone_data.open_window_attr
 
+        elif self.zone_variable == "open window detected":
+            self._state = self._tado_zone_data.open_window_detected
+
```

Comment:
```suggestion
```

Reply:
I had clicked to commit suggestions. Anything else I should do?

Reply:
Looks like they got applied out of order so there is no longer any code for github to run its suggestion engine on.